metadata/src/main/java/edu/unc/lib/dl/util/DepositStatusFactory.java

```diff
@@ -89,7 +88,6 @@ public void addManifest(String depositUUID, String value) {
 	public void save(String depositUUID, Map<String, String> status) {
 		try (Jedis jedis = getJedisPool().getResource()) {
 			jedis.hmset(DEPOSIT_STATUS_PREFIX + depositUUID, status);
```

Comment:
Could you also remove the declaration of DEPOSIT_SET from the constants class since it is no longer used anywhere?